### PR TITLE
Fixed clockIn and clockOut bugs

### LINKED ISSUE #171: Inconsistency in format of commands
**Please describe the problem**
Some commands are in Java camel case, e.g. `showMainMenu`, `clockIn` while others are in all lower case `listmenu`, `addorder`

**Describe the improvement you'd like to suggest**
All commands should be in the same format to make it easier for the user to use the app.

<hr>

**Reported by:** @nianfei97
**Severity:** `Medium`

<sub>[original: nusCS2113-AY1819S1/pe-1#731]</sub>

### PR DESCRIPTION
**Pull request by kianhong95**
Both commands now catch index out of bounds exception.
Changed naming to clocking and clockout which is standardised with the other commands.

Resolves #171

### CODE REVIEW

File: src/seedu/addressbook/commands/employee/EmployeeClockInCommand.java [hunk 1]
```diff
@@ -54,17 +56,22 @@ private Attendance createNewAttendance(Attendance oldAttendance) {
...
-        Attendance newAttendance = createNewAttendance(oldAttendance);
+            Attendance newAttendance = createNewAttendance(oldAttendance);
 
-        rms.updateAttendance(oldAttendance, newAttendance);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, name, this.currentDate));
+            rms.updateAttendance(oldAttendance, newAttendance);
+            return new CommandResult(String.format(MESSAGE_SUCCESS, name, this.currentDate));
+        }
+        catch (IndexOutOfBoundsException ie) {
+            return new CommandResult(Messages.MESSAGE_EMPLOYEE_NOT_IN_RMS);
```
Comment (px1099) [inline src/seedu/addressbook/commands/employee/EmployeeClockInCommand.java:73]: Error message should not be `MESSAGE_EMPLOYEE_NOT_IN_RMS`, it should be `MESSAGE_INVALID_EMPLOYEE_DISPLAYED_INDEX`

File: src/seedu/addressbook/commands/employee/EmployeeClockOutCommand.java [hunk 2]
```diff
@@ -52,17 +54,21 @@ private Attendance createNewAttendance(Attendance oldAttendance) {
...
 
-        Attendance newAttendance = createNewAttendance(oldAttendance);
+            Attendance newAttendance = createNewAttendance(oldAttendance);
 
-        rms.updateAttendance(oldAttendance, newAttendance);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, name, this.currentDate));
+            rms.updateAttendance(oldAttendance, newAttendance);
+            return new CommandResult(String.format(MESSAGE_SUCCESS, name, this.currentDate));
+        } catch (IndexOutOfBoundsException ie) {
+            return new CommandResult(Messages.MESSAGE_EMPLOYEE_NOT_IN_RMS);
```
Comment (px1099) [inline src/seedu/addressbook/commands/employee/EmployeeClockOutCommand.java:71]: Same as above

**Review by px1099 — DISMISSED** [review]

File: src/seedu/addressbook/commands/employee/EmployeeClockOutCommand.java (hunk 2 above)
Comment (kianhong95) [reply src/seedu/addressbook/commands/employee/EmployeeClockOutCommand.java:71]: Edited.

**Comment by px1099** [comment]
Wrong message
The one I am referring to is the existing Message.MESSAGE_INVALID_EMPLOYEE_DISPLAYED_INDEX
The point being the content of the error message is not suitable, it is not about the name of constant

**Review by px1099 — APPROVED** [review]

**Comment by px1099** [comment]
Resolve #165

**Pull request merged**